spotbugs/etc/messages.xml

```diff
@@ -8785,11 +8785,29 @@ Using floating-point variables should not be used as loop counters, as they are
       </p>]]>
     </Details>
   </BugPattern>
+
+  <BugPattern type="IAOM_DO_NOT_INCREASE_METHOD_ACCESSIBILITY">
+    <ShortDescription>Accessibility of an overridden or hidden method was increased.</ShortDescription>
+    <LongDescription>Accessibility of an overridden or hidden method {1} was increased from {2} to {3}.</LongDescription>
+    <Details>
+      <![CDATA[<p>
+        Increasing the accessibility of overridden or hidden methods permits a malicious subclass to offer wider
```

Comment:
How many programs might actually be used in a context where malicious subclasses are a realistic threat? Applets are long out of fashion...

Reply:
In my opinion, when developers are creating methods with an explicit visibility modifier, they do it for a reason. If a subclass overrides this behavior, then it ignores the intentions of the creator of the base class which, can lead to bugs and code which is hardly interpretable. Anyway, it is a very bad practice.